daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/SeparatedSequenceParsers.scala

```diff
@@ -146,8 +143,24 @@ final class ScalarOrderedSeparatedSequenceChildParser(
 
 }
 
-final class RepOrderedExactlyNSeparatedSequenceChildParser(
-  childParser: Parser,
+final class ScalarOrderedSeparatedSequenceChildParser(childParser: Parser,
+  srd: SequenceRuntimeData,
+  trd: TermRuntimeData,
+  sep: Parser,
+  spos: SeparatorPosition,
+  ssp: SeparatorSuppressionPolicy)
+  extends ScalarOrderedSeparatedSequenceChildParserBase(childParser, srd, trd, sep, spos, ssp)
+
+final class PotentiallyTrailingGroupSeparatedSequenceChildParser(childParser: Parser,
```

Comment:
As above. This class could be eliminated. See corresponding comments about Unparser side.